src/components/RenderError.js

```diff
@@ -22,11 +25,33 @@ type Props = {
   children?: *,
 }
 
-class RenderError extends PureComponent<Props, { isHardResetting: boolean }> {
+class RenderError extends PureComponent<
+  Props,
+  { isHardResetting: boolean, isHardResetModalOpened: boolean },
+> {
   state = {
     isHardResetting: false,
+    isHardResetModalOpened: false,
   }
 
+  handleOpenHardResetModal = () => this.setState({ isHardResetModalOpened: true })
+  handleCloseHardResetModal = () => this.setState({ isHardResetModalOpened: false })
+
+  handleHardReset = async () => {
```

Comment:
you forgot to remove the other `handleHardReset` method :p

Reply:
I certainly did 🙄and neither lint nor something else complained..